Create INSTALL.md

aplerdal opened the pull request:
Created an install guide with all the information you sent me.

aplerdal committed:
Create INSTALL.md

jellees commented on INSTALL.md:
````diff
@@ -0,0 +1,43 @@
+export DEVKITARM=/opt/devkitpro/devkitARM
+```
+### Installing tools
+Donwload the main tools from [here](https://cdn.discordapp.com/attachments/960983413890908220/960983654220308502/tools.7z?ex=65e771c5&is=65d4fcc5&hm=47aa4fe20beb4d9ab203832eeec680b8b3f70dc6c92eab55a062ce61815490a9&).
````
It's not a good idea to host tools on discord and then link them here. Instead the user should probably just install those tools themself. Agbcc can be found here https://github.com/pret/agbcc. As for gbafix, I am not sure where I got that from but it is also part of devkitpro, so perhaps we should just use that instead.

aplerdal committed:
Update INSTALL.md

- Fix spelling mistakes
- Improve clarity
- Remove the discord link

aplerdal committed:
Fix install path error

aplerdal committed:
use gbafix from devkitpro

Was installed during INSTALL.md instructions

aplerdal committed:
Add -j to installation

aplerdal commented:
I fixed everything in the INSTALL.md file and tested it (WSL, Debian). I also switched the makefile to just use gbafix from the path, as it should be installed when following the instructions I wrote. Let me know if there is anything else I still need to fix!

jellees reviewed (approved):
Looks good now!
